app/tests/test_main.py

```diff
@@ -1,6 +1,6 @@
 import pytest
 
-from ..main import main, Genre, Actor
+from ..main import main, Genre, Actors
```

Comment:
Typo in README.md was fixed, use `Actor` instead of `Actors`.

Reply:
fixed)